src/aap_eda/settings/default.py

```diff
@@ -781,15 +781,6 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
     "ACTIVATION_DB_HOST", "host.containers.internal"
 )
 
-_DEFAULT_PG_NOTIFY_DSN = (
-    f"host={ACTIVATION_DB_HOST} "
-    f"port={DATABASES['default']['PORT']} "
-    f"dbname={DATABASES['default']['NAME']} "
-    f"user={DATABASES['default']['USER']} "
-    f"password={DATABASES['default']['PASSWORD']}"
-)
-
-PG_NOTIFY_DSN = settings.get("PG_NOTIFY_DSN", _DEFAULT_PG_NOTIFY_DSN)
```

Comment:
Right now the user is able to override this setting through EDA_PG_NOTIFY_DSN. I believe it is very unlikely that someone has overrided this, but, it's still a breaking change if we just remove it. 

So once there is a decision about if want to continue supporting it, isn't? 
Should be honor it when we create the eda_credential?

Reply:
@Alex-Izquierdo This is not exposed via any of our installers so we can pretty safely remove it. We only have a single database support currently when we have a separate Database for EventStream we will have to revisit this.